### DISCUSSION TITLE
Oops w/ Oauth, no error displayed

### DISCUSSION THREAD

**Discussion by andymelichar** [requarks/wiki#5943] [Error / Bug Report]
Setting up the Oauth2 provider, when I try and log in I get all the way through the auth process with the SSO provider, and when I am redirected to my wiki.js instance, I get a 500 error / Oops, something went wrong message, but there is no specific error displayed. 

I've tried setting my log level to debug, but it doesn't seem to have any effect - nothing shows up in the logs.

**Comment by DavidOhsieAtTurbo** [comment]
Same problem here.  There appears to be very little debug logging in the code, so the flag may be taking effect, but no debug logging is being triggered.  But I'm surprised that a 500 error doesn't generate anything in the log.  It would be nice to get the underlying passport authentication system to log, but I don't know if that is possible.
